src/ebay-fake-menu-button/menu-button-icon.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react'
+import { EbayIcon, Icon } from '../index'
+
+type Props = {
+    name: Icon;
+    className?: string;
+}
+
+const EbayMenuButtonIcon: FC<Props> = ({
+    name,
+    className
+}) => (
+    <EbayIcon
+        name={name}
+        className={className}
+    />
+)
```

Comment:
Any specific reason why do we have to wrap EbayIcon as EbayMenuButtonIcon? This looks unnecessary since:

- EbayMenuButtonIcon does not add anything, not even additional class name
- Not consistent with other component, for example EbayButton with custom icon:

```jsx
<EbayButton variant="destructive" aria-label="Destructive button">
  <EbayIcon name="delete" />
  <span>Button with icon</span>
</EbayButton>
```
- On Marko, the attribute `@icon` contains ebay-icon, also without additional wrapping: https://github.com/eBay/ebayui-core/blob/4c531971dc0cf25ffec34814feb3f83d31d20b19/src/components/ebay-fake-menu-button/examples/09-icon-with-text/template.marko#L13-L15

I think we can live with just EbayIcon inside EbayFakeMenu